app/views/act/participants/log_in_to_new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Chyba - nepřihlášený uživatel</h1>
+Aby sis mohl založit uživatelský účet, musíš se nejprve
+<%= link_to "přihlásit", "https://pikomat.mff.cuni.cz/users/sign_in" %>.<br>
```

Comment:
nedavej url natvrdo. pak to to v tep-dev- nebude fungovat.

Reply:
Jo, dávat to natvrdo je samozřejmě obecně špatně (pokud je alternativa). Snad by tohle měl být jediný výskyt. (jak je to řešené ttou knihovnou ```devise```, tak jsem nevěděl, jak se k té cestě na přihlašování dostat, tak jsem to odložil na později a zapomněl jsem se k tomu vrátit. Ale už to je opravené)